### DISCUSSION TITLE
Network folder

### DISCUSSION THREAD

**Discussion by PM242** [rejetto/hfs#668] [General]
Hello,

If I run hfs.exe directly from my folder c:\hfs release is 0.53.0 rc16 and from http://localhost/~/admin/#/fs I can access to C: drive but also the network drive like Z: and X:
But if I try to install hfs as windows service with the following command:

run npm -g i hfs
run npx qckwinsvc2 install name="HFS" description="HFS" path="%APPDATA%\npm\node_modules\hfs\src\index.js" args="--cwd %HOMEPATH%\.hfs" now

When I open the console http://localhost/~/admin/#/f  I can see only C: drive but not the shared drive like Z: and X:, someone have and idea why I cannot see shared drive (Z: & X;)?
Other thing, there is a difference between the release, hfs.exe on c:\hfs is  release is 0.53.0 rc16 but If I use the procedure above (run npm -g i hfs) release is 0.52.9, I don't know if the difference between these two release can explain my problem with network drive not avalaible? or maybe it's an another problem?

My goal it's to use hfs as windows and access to my network drive, I hope that it's possible.

Thank you in advance for your assistance.

**Comment by rejetto** [comment]
i think your problem is not related to the version, but to the user.
by default services are run with a different user.
i checked qckwinsvc2, and it doesn't have an option to specify the user.
access the list of services in Windows, and change the user for hfs, manually.

the reason you are not getting 0.53 with npm is that it is still in beta version, but you can do it with `npm -g i hfs@beta`

**Comment by PM242** [comment]
Thank you for reply, I changed account with administrator privilege to start windows services, same problem I cannot see the network drive, I don't know if thre is solution to solve my problem.

**Comment by SanokKule** [comment]
This might be related https://woshub.com/how-to-access-mapped-network-drives-from-the-elevated-apps/

**Comment by PM242** [comment]
Thank you for the link.  I can access to X:\ with command >dir x: without restriction, I added EnableLinkedConnections in the registry as mentionned but no change, I cannot see the network drive.

**Reply by TEA-Time1** [reply]
I just tried that registry entry and it worked (thanks @SanokKule!), so I recommend double checking that you did it right, including restarting after making that change like it says.

**Reply by PM242** [reply]
Thank you so mutch @TEA-Time1, it's works fine if I specify UNC path! Just a last question in this case (UNC path) it's not necessary to add key "EnableLinkedConnections" in the registry?

**Reply by TEA-Time1** [reply]
Yup, that's exactly how I tested it. That setting is necessary for a UNC mapped drive to be accessible in an elevated process.

**Reply by rejetto** [reply]
sorry @PM242, instead of replying i edited your post. I reverted it to the original text.

my comment was:
> If you can't solve, consider accessing those spaces specifyin the UNC path.

**Comment by rejetto** [comment]
anyway, if you google for it, you find it is a known problem, of Windows services in general
